code/modules/surgery/organs/tongue.dm

```diff
@@ -24,61 +25,67 @@
 	. = ..()
 	languages_possible = languages_possible_base
 
-/obj/item/organ/tongue/get_spans()
-	return list()
-
-/obj/item/organ/tongue/proc/TongueSpeech(var/message)
-	return message
+/obj/item/organ/tongue/proc/handle_speech(datum/source, list/speech_args)
 
 /obj/item/organ/tongue/Insert(mob/living/carbon/M, special = 0)
 	..()
 	if(say_mod && M.dna && M.dna.species)
 		M.dna.species.say_mod = say_mod
+	if (modifies_speech)
+		RegisterSignal(M, COMSIG_MOB_SAY, .proc/handle_speech)
+	M.UnregisterSignal(M, COMSIG_MOB_SAY)
 
 /obj/item/organ/tongue/Remove(mob/living/carbon/M, special = 0)
 	..()
 	if(say_mod && M.dna && M.dna.species)
 		M.dna.species.say_mod = initial(M.dna.species.say_mod)
+	UnregisterSignal(M, COMSIG_MOB_SAY, .proc/handle_speech)
+	M.RegisterSignal(M, COMSIG_MOB_SAY, /mob/living/carbon/.proc/handle_tongueless_speech)
```

Comment:
Do you know if overriding handle_tongueless_speech still works? I'm curious if something like `M.type.proc/handle_tongueless_speech` is required

Reply:
```
/mob/living/carbon/monkey/handle_tongueless_speech(datum/source, list/speech_args)
	speech_args[SPEECH_MESSAGE] = "Lol i have no tongue"
```
``` 
ADMIN LOG: Naksuasdf/(Ree-Chulz) has removed /obj/item/organ/tongue/lizard from Naksuasdf/(Ree-Chulz)(?) (FLW)
Ree-Chulz chimpers, "Lol i have no tongue"
```